setup.md

```diff
@@ -226,69 +271,114 @@ First, you should set your identity.
 One of the most important things that version control like Git does is to keep track of who changes what.
 This helps repository maintainers coordinate the efforts of all the people who contribute to the project.
 Most importantly, it makes it easier to figure out who to blame when something goes wrong.
-You can provide Git your name and contact information with the following commands:
+You can provide git your name and contact information with the following commands:
+
+> ## Configuring git
+> In the command below, you do not need to put your name or your email address in all caps.
+{: .callout}
 
 ~~~
-$ git config --global user.name "YOUR NAME"
-$ git config --global user.email "YOUR_EMAIL"
+git config --global user.name "YOUR_FIRSTNAME YOUR_LASTNAME"
+git config --global user.email "YOUR_EMAIL_ADDRESS"
 ~~~
-{: .language-bash}
-You should next set the default branch name you will work on. Branches will be explained in detail later.
-This configuration setting makes the default branch name match the default branch name on GitHub
+{: .bash}
+
+Next, you will need to set the name of the default branch git uses.
+The following command will set your default branch name to be "main"
 
 ~~~
 git config --global init.defaultBranch main
 ~~~
-{: .language-bash}
+{: .bash}
```

Comment:
Is this a necessary update to the template usage? Do we need to update the rest of the docs too?

Reply:
I thought you were talking about `git` initially, but I think you're refering to `{: .language-bash}` vs {: .bash}`. No, no need to update. The two are equivalent. I usually like using ``{: .language-bash}` better because it's more verbose, but the behavior is the same.

Reply:
👍 
If you like `{: .language-bash}`, why change it here? 😛

Reply:
I forgot :)